cellxgene_schema_cli/tests/test_schema_compliance.py

```diff
@@ -281,16 +281,31 @@ def test_assay_ontology_term_id(self):
 
     def test_cell_type_ontology_term_id(self):
         """
-        cell_type_ontology_term_id categorical with str categories. This MUST be a CL term.
+        cell_type_ontology_term_id categorical with str categories. This MUST be a CL term, and must NOT match forbidden
```

Comment:
Do we have a locally glossary somewhere for these acronyms(CL)?

Reply:
I don't think its an acronym, its a prefix used for cell ontology terms. There's a glossary in the user-facing [schema](https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/4.0.0/schema.md#required-ontologies) documentation